01-JS-DOM-APIs/normalize.css

```diff
@@ -1,11 +1,5 @@
-/*! normalize.css v4.1.1 | MIT License | github.com/necolas/normalize.css */
-
-/**
- * 1. Change the default font family in all browsers (opinionated).
- * 2. Prevent adjustments of font size after orientation changes in IE and iOS.
- */
-
 html {
+<<<<<<< HEAD:01-JS-DOM-APIs/normalize.css
```

Comment:
Here you have a conflict. You should fix it. Look up for conflict solving in Internet